run-checks

```diff
@@ -150,10 +150,18 @@ if [ "$STATIC" = 1 ]; then
     echo Checking docs
     ./mdlint.py ./*.md docs/*.md
 
+    # XXX: obsolete
     if [ -n "${TRAVIS_PULL_REQUEST:-}" ] && [ "${TRAVIS_PULL_REQUEST:-}" != "false" ]; then
-        echo Checking pull request summary
+        echo "Checking pull request summary"
         ./check-pr-title.py "$TRAVIS_PULL_REQUEST"
     fi
+    # XXX: is there a better way on GH?
+    if [ -n "${GITHUB_EVENT_PATH:-}" ]; then
+        set -x
+        echo "Checking pull request summary"
+        PR_TITLE="$(jq -r .pull_request.title $GITHUB_EVENT_PATH)"
```

Comment:
You can certainly implement this entirely in javascript and regexp on an action. Perhaps we should go shopping for some off-the-shelf action that does just that?

Reply:
Works for me

Reply:
I'd say if this works, we should just merge this and leave using something not-bespoke to a TODO